lasagne/tests/test_utils.py

```diff
@@ -1,8 +1,8 @@
-from mock import Mock
-import pytest
 import numpy as np
+import pytest
 import theano
 import theano.tensor as T
+from mock import Mock
```

Comment:
Please configure your editor to not reformat existing code! Just leave imports as they were.